tka/tailchonk.go

```diff
@@ -189,6 +190,21 @@ updateLoop:
 	return nil
 }
 
+// BackdateCommitTimes will move all the commit times back by the
+// specified duration.
+//
+// This is only for use in tests, and will panic if called in non-test code.
+func (c *Mem) BackdateCommitTimes(d time.Duration) {
```

Comment:
This is a bit crude - what do you think about adding `timeNow func() time.Time` to `Mem`, which we will be able to override in tests? It's a common pattern we use elsewhere.